.github/dependabot.yml

```diff
@@ -18,7 +18,7 @@ updates:
     versions:
     - ">= 0"
 - package-ecosystem: npm
-  directory: "/api_tests"
+  directory: "/e2e_tests"
```

Comment:
The reason I called them e2e tests is because I see them as e2e tests for _comit-rs_. Fits well actually, not sure if the reduction down to `tests` is that much better?

Reply:
Let's continue the discussion here: https://github.com/comit-network/comit-rs/pull/3199#discussion_r492433899